react/BottomSheet/BottomSheet.jsx

```diff
@@ -154,11 +154,10 @@ const BottomSheet = ({
     ANIMATION_DURATION,
     isClosable && isBottomPosition
   )
+  const innerContentHeight = innerContentRef?.current?.offsetHeight ?? 0
```

Comment:
When fixing bug in a cozy library, it should be good to either add a test, or update doc inside a Readme.md